router.go

```diff
@@ -66,10 +66,10 @@ func Router(
 	kissMyRankHandler *KissMyRankHandler,
 ) http.Handler {
 	r := chi.NewRouter()
-
+	compressor := middleware.Compress(5)
```

Comment:
it looks like middleware.Compress is deprecated too! I think `middleware.NewCompressor` is the latest and greatest :-)

Reply:
I don't believe Compress is deprecated as it's not mentioned anywhere. DefaultCompressor had a comment that it was deprecated before it was removed. Compress seems to just be the new "convenience method" as a wrapper to NewCompressor.

I've fixed up the spacing.

Reply:
Ah ok, looks like a versioned install of chi on my machine was outdated. They un-deprecated `Compress` in a recent commit, so I'll accept this :grin: